src/main/java/de/tum/in/www1/artemis/web/rest/StatisticsResource.java

```diff
@@ -27,146 +27,18 @@ public StatisticsResource(StatisticsService service) {
     }
 
     /**
-     * GET management/statistics/users : get the amount of logged in user in the last "span" days.
+     * GET management/statistics/data : get the graph data in the last "span" days in the given period.
      *
-     * @param span the period of which the amount should be calculated
-     * @return the ResponseEntity with status 200 (OK) and the amount of users in body, or status 404 (Not Found)
+     * @param span the spantime of which the amount should be calculated
+     * @param periodIndex an index indicating which time period, 0 is current week, -1 is one week in the past, -2 is two weeks in the past ...
+     * @param graphType the type of graph the data should be fetched
+     * @return the ResponseEntity with status 200 (OK) and the data in body, or status 404 (Not Found)
      */
-    @GetMapping("management/statistics/users")
+    @GetMapping("management/statistics/data")
     @PreAuthorize("hasAnyRole('INSTRUCTOR', 'ADMIN')")
```

Comment:
As this is an admin-only feature, I'd restrict this endpoint to Admins only.

Reply:
You are right, adjusted that in [this](https://github.com/ls1intum/Artemis/pull/2511/commits/29286d4a70141478ca0c1481b0295e4ef7abc44b) commit